citi-onboarding/virtu-onepage
# Feature/contact nodemailer

### What I did
- Added nodemailer dependency to the project
- Made new post route to handle the email send
- Created transporter to send email
- Added new information's to the dotenv file, so that it works with nodemailer
- Integrated the route with the frontend

### How to test
1. Enter server folder with `cd server`
2. Execute the command `yarn` to download the server modules
3. Execute the command `yarn start` to start the server
4. Execute the command `cd ../client` and then `yarn` to download the client modules
5. Execute the command `yarn start` to start the client
6. Open your browser on `http://localhost:3001/admin` to view keystone admin
7. Log in with the credentials available at Notion
8. Open another browser page on `http://localhost:3000` to see the client page
9. Add the dotenv available at Notion
10. Try to send the form and see if the email gets send 🙃 

Resolves #34
